src/main/java/io/github/guild/application/controller/guild/GuildController.java

```diff
@@ -29,6 +33,9 @@
     private final CommandHandler<CreateGuildCommand> createGuildCommandHandler;
     private final CommandHandler<DeleteGuildCommand> deleteGuildCommandHandler;
 
+    @Autowired
+    private GuildService guildService;
```

Comment:
There is no need to use @Autowired... You already injected 2 Command Handlers without @Autowired. Use the same approach